CONTRIBUTING.md

```diff
@@ -160,6 +160,44 @@ const MyComponent: FC<MyComponentProps> = ({ prop1, prop2... }) => (
 export default MyComponent;
 ```
 
+#### How to I18n my component
+
+In Node.js website we use [react-intl](https://formatjs.io/docs/react-intl/) to handle internationalization.
+
+There are two type of components, one use content write in markdown and one use react-intl formatted message.
```

Comment:
LGTM to me with some typos:

There are two **types** of components, one **uses** content write in markdown and **the other uses** react-intl formatted message.